homeassistant/components/netgear_lte/sensor_types.py

```diff
@@ -10,28 +10,17 @@
     SENSOR_SMS: "unread",
     SENSOR_SMS_TOTAL: "messages",
     SENSOR_USAGE: "MiB",
-    "radio_quality": "%",
-    "rx_level": "dBm",
-    "tx_level": "dBm",
-    "upstream": None,
```

Comment:
Ok, it's this part that changes the names of some sensors, right?

Reply:
Since we have unique_id, it's only the new entities that will change name and entity_id. Existing registered entities will not be affected.

Reply:
I think you got that right. So when removing the `sensor:` subsection from this existing configuration:
```yaml
netgear_lte:
  - host: IP_ADDRESS
    password: SECRET
    sensor:
      monitored_conditions:
        - usage
        - radio_quality
```
... the `sensor.netgear_lte_usage` entity is retained but is now created automatically.

The `sensor.netgear_lte_radio_quality` entity is no longer added but a new, disabled `sensor.netgear_lte_wwanadv_radioquality` entity with the same information is added instead.

Reply:
The existing entities in the registry, will they also be disabled by default or will they be added?

Reply:
Existing entities will not be disabled because `entity_registry_enabled_default` only takes effect when the entity is added to the registry.

Reply:
Thanks!